### PR TITLE
Remove restarts in patch tests

### PR DESCRIPTION
**Pull request by sdkottegoda**

### CODE REVIEW

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/car/deployment/test/ESBJAVA4116CAppArtifactDeploymentFailureDueToNameTestCase.java [hunk 1]
```diff
@@ -23,34 +23,20 @@
 import org.testng.annotations.Test;
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/car/deployment/test/ESBJAVA4116CAppArtifactDeploymentFailureDueToNameTestCase.java]: the license should be updated.

Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/car/deployment/test/ESBJAVA4116CAppArtifactDeploymentFailureDueToNameTestCase.java]: License should be updated with the created year

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/endpoint/test/ESBJAVA3340QueryParamHttpEndpointTestCase.java [hunk 2]
```diff
@@ -50,7 +50,7 @@ public void testPassParamsToEndpoint() throws Exception {
         logViewerClient.clearLogs();
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/endpoint/test/ESBJAVA3340QueryParamHttpEndpointTestCase.java]: License should be updated

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/jms/transport/test/JMSBrokerStartupTestCase.java [hunk 3]
```diff
@@ -31,12 +31,12 @@
     @BeforeTest(alwaysRun = true)
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/jms/transport/test/JMSBrokerStartupTestCase.java]: Update license

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/mediators/rule/ESBJAVA2506RuleFetchFromRegistryFailsForTheFirstTime.java [hunk 4]
```diff
@@ -23,36 +23,21 @@
 import org.testng.annotations.Test;
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/mediators/rule/ESBJAVA2506RuleFetchFromRegistryFailsForTheFirstTime.java]: License needs to be updated with the year created.

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4331MissingJSONEmptyArrayNHTTPTransport.java [hunk 5]
```diff
@@ -42,11 +42,8 @@ public void setEnvironment() throws Exception {
         serverConfigurationManager =
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4331MissingJSONEmptyArrayNHTTPTransport.java]: follow Apache license format

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4940CharacterEncodingRemovalTestCase.java [hunk 6]
```diff
@@ -29,6 +29,7 @@
 import org.wso2.carbon.logging.view.stub.types.carbon.LogEvent;
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4940CharacterEncodingRemovalTestCase.java]: Follow apache license format

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/mtom/test/ESBJAVA4909MultipartRelatedTestCase.java [hunk 7]
```diff
@@ -31,7 +31,6 @@
  * Test case check whether attachment is received by the client
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/mtom/test/ESBJAVA4909MultipartRelatedTestCase.java]: please include license

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA4883SynapseHandlerBlockingCallTestCase.java [hunk 8]
```diff
@@ -43,8 +43,6 @@
     public void init() throws Exception {
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA4883SynapseHandlerBlockingCallTestCase.java]: License needs to be updated with the year created and the license format.

File: integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/rest/query_params_api.xml [hunk 9]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/rest/query_params_api.xml]: Add license to the life

File: integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/vfsTransport/ESBJAVA4770/axis2.xml [hunk 10]
```diff
@@ -257,7 +257,7 @@
         <parameter name="non-blocking" locked="false">true</parameter>
```
Comment (srsharon) [inline integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/vfsTransport/ESBJAVA4770/axis2.xml]: License needs to be updated with the year created.

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/car/deployment/test/ESBJAVA4116CAppArtifactDeploymentFailureDueToNameTestCase.java (hunk 1 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/car/deployment/test/ESBJAVA4116CAppArtifactDeploymentFailureDueToNameTestCase.java]: It's already there :)

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/mediators/rule/ESBJAVA2506RuleFetchFromRegistryFailsForTheFirstTime.java (hunk 4 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/mediators/rule/ESBJAVA2506RuleFetchFromRegistryFailsForTheFirstTime.java]: It's already there :)

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA4883SynapseHandlerBlockingCallTestCase.java (hunk 8 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/ESBJAVA4883SynapseHandlerBlockingCallTestCase.java]: It's already there :)

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/endpoint/test/ESBJAVA3340QueryParamHttpEndpointTestCase.java (hunk 2 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/endpoint/test/ESBJAVA3340QueryParamHttpEndpointTestCase.java]: done :)

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/jms/transport/test/JMSBrokerStartupTestCase.java (hunk 3 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/jms/transport/test/JMSBrokerStartupTestCase.java]: done

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4331MissingJSONEmptyArrayNHTTPTransport.java (hunk 5 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4331MissingJSONEmptyArrayNHTTPTransport.java]: done

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4940CharacterEncodingRemovalTestCase.java (hunk 6 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/json/test/ESBJAVA4940CharacterEncodingRemovalTestCase.java]: done

File: integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/mtom/test/ESBJAVA4909MultipartRelatedTestCase.java (hunk 7 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/nhttp/transport/mtom/test/ESBJAVA4909MultipartRelatedTestCase.java]: added

File: integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/rest/query_params_api.xml (hunk 9 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/rest/query_params_api.xml]: added

File: integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/vfsTransport/ESBJAVA4770/axis2.xml (hunk 10 above)
Comment (sdkottegoda) [reply integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/synapseconfig/vfsTransport/ESBJAVA4770/axis2.xml]: done

**Pull request merged**